src/bin/travis/testCentaurTes.sh

```diff
@@ -31,26 +31,48 @@ printTravisHeartbeat
 set -x
 set -e
 
+WORKDIR=$(pwd)
+
 sbt assembly
 CROMWELL_JAR=$(find "$(pwd)/target/scala-2.11" -name "cromwell-*.jar")
 TES_CENTAUR_CONF="$(pwd)/src/bin/travis/resources/tes_centaur.conf"
 git clone https://github.com/broadinstitute/centaur.git
 cd centaur
 git checkout ${CENTAUR_BRANCH}
-cd ..
+cd $WORKDIR
+
 
-TES_CONF="$(pwd)/src/bin/travis/resources/tes.conf"
-git clone https://github.com/ohsu-comp-bio/funnel.git
-cd funnel
-git checkout 7adbf0b
+FUNNEL_CONF="$(pwd)/src/bin/travis/resources/funnel.conf"
+wget https://storage.googleapis.com/golang/go1.8.1.linux-amd64.tar.gz
+tar xfz go1.8.1.linux-amd64.tar.gz
+export GOROOT=$WORKDIR/go
+mkdir go-lib
+export GOPATH=$WORKDIR/go-lib
+go get github.com/ohsu-comp-bio/funnel
+cd $GOPATH/src/github.com/ohsu-comp-bio/funnel
+git checkout 2995adb
 make
-cd ..
+cd $WORKDIR
 mkdir logs
-nohup funnel/bin/tes-server -config ${TES_CONF} > logs/tes.log 2>&1 &
+nohup $GOPATH/bin/funnel server --config ${FUNNEL_CONF} > logs/funnel.log 2>&1 &
 
 
 # All tests use ubuntu:latest - make sure it's there before starting the tests 
 # because pulling the image during some of the tests would cause them to fail 
 # (specifically output_redirection which expects a specific value in stderr)
 docker pull ubuntu:latest
-centaur/test_cromwell.sh -j"${CROMWELL_JAR}" -c${TES_CENTAUR_CONF} -elocaldockertest
+
+# The following tests are skipped:
+#
+# non_root_specified_user:   TES doesn't support switching users in the image
+# write_lines_files:         all inputs are read-only in TES
+# lots_of_inputs:            Funnel mounts in each input separately, this task surpasses the docker limit for volumes
+# call_cache_capoeira_local: fails on task 'read_files_without_docker' since the 'docker' runtime key is required for this backend
```

Comment:
Hmm, that's a shame... perhaps we could add a `call_cache_capoeira_tes` with it removed?